DATA/production/qc-async/mid.json

```diff
@@ -3,23 +3,18 @@
     "config": {
       "database": {
         "implementation": "CCDB",
-        "host": "ali-qcdb.cern.ch:8083",
-        "username": "not_applicable",
-        "password": "not_applicable",
-        "name": "not_applicable"
+            "host": "ccdb-test.cern.ch:8080"
```

Comment:
I cannot immediately see what happened here, but it doesn't seem to be a normal comma at the end of this line!? Not sure, maybe with your editor some special character was inserted here. The same is true for some other lines below.

Reply:
yes I suppose that it comes from my editor it is not normal.
How can I remove this?

Reply:
What editor are yo using? Are you using any special encoding?